libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -18,7 +19,7 @@ const AP_Param::GroupInfo AP_Camera::var_info[] = {
     // @Param: TRIGG_TYPE
     // @DisplayName: Camera shutter (trigger) type
     // @Description: how to trigger the camera to take a picture
-    // @Values: 0:Servo,1:Relay
+    // @Values: 0:Servo,1:Relay, 2:GoPro 3/4 in mavlink Solo Gimbal
```

Comment:
Took me a while to work out what three-quarters of a was :-)

Perhaps "mavlink" is unnecessary here?

Reply:
Ya I think  I can probably just make it say `GoPro in Solo Gimbal`.  The 3 and 4 is implied since those are the only ones that fit in the gimbal. And mavlink is implied since that's what the gimbal uses.  I'll change all of those.